compose-files/pleuston.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.1'
+services:
+  pleuston:
+    image: oceanprotocol/pleuston:${OCEAN_VERSION:-stable}
+    networks:
+      backend:
+        ipv4_address: 172.15.0.18
+    ports:
+    - 3000:3000
+    depends_on:
+    - keeper-contracts
```

Comment:
I think Pleuston will be unhappy if it can't connect to `aquarius` and `brizo`.

It might also want some web3 things to be around (e.g. `keeper-contracts`, `parity-node` or `secret-store`) but I'm not sure. I guess the "parity node" and "secret store" could be remote ones (i.e. not part of the local Docker Compose set of services), but I guess we're not supporting that in docker-images yet.

@jernejpregelj or @kremalicious you know best what Pleuston needs.

Reply:
Pleuston needs the compiled contract artifacts from `keeper-contracts`, they need to be present before the build script runs. (At least this was the case the last time I checked, I'm not aware if that might not be needed anymore.)

Reply:
`Pleuston` needs access over localhost since it runs in browser and not inside docker network.
Current requirements are:
```
      NODE_HOST: localhost
      AQUARIUS_HOST: localhost
      BRIZO_HOST: localhost
```
It does not need network access to `keeper-contracts` but it still needs to wait for that container to build the contracts as I understand.

Reply:
Why Pleuston can not connect to remote aquarius/brizo components? Does this apply to the docker-compose environment, or to any environment?